src/server.py

```diff
@@ -535,8 +547,11 @@ def buildbot_name(platform, buildtype, jobname, branch):
     if not branch:
         branch = "%s"
 
+    if jobname.startswith(('tp5o', 'chromez', 'dromaeojs', 'other', 'g1', 'g2', 'xperf', 'svgr')):
+        buildtype = 'talos'
+
```

Comment:
@jmaher On thinking about this, why is buildtype set here for talos? buildtype is already passed to this function, is there a reason to believe that the buildtype is not correctly set?